examples/tests/ble/ble_test_tx_power/main.c

```diff
@@ -25,13 +25,13 @@ int main(void) {
 
     if (is_valid) {
       err = ble_set_tx_power(i);
-      if (err < TOCK_SUCCESS) {
+      if (err < RETURNCODE_SUCCESS) {
         printf("Test failed \t power_level %d could not be configured\r\n", i);
         return 0;
       }
     } else {
       err = ble_set_tx_power(i);
-      if (err >= TOCK_SUCCESS) {
+      if (err >= RETURNCODE_SUCCESS) {
```

Comment:
```suggestion
      if (err == RETURNCODE_SUCCESS) {
```

It should no longer be possible for this function to return a value greater than 0 (nit)

Reply:
I think I fixed this correctly.

Reply:
I don't think so. This test is actually verifying that if you try to set an invalid power level an error is returned

Reply:
Ok, gotcha, I didn't understand the printf comment. Fixed now.